src/app/[identifier]/page.tsx

```diff
@@ -28,16 +31,12 @@ const renderSkeletonHeader = () => (
   </Card>
 );
 
-const copyToClipboard = (text: string) => {
-  navigator.clipboard.writeText(text).then(() => {
-  }).catch(err => {
-  });
-};
-
 export default function Page({ params }: ResponseProps) {
   const identifier = decodeURIComponent(params.identifier);
   const fid: number | null = isNumeric(identifier) ? Number(identifier) : null;
   const hash = fid ? null : identifier;
+
+  const { copied, copy } = useClipboard();
```

Comment:
convenient hook to flip between copied state and back

Reply:
this is way cleaner lmao nice